api/src/services/graphql.ts

```diff
@@ -2104,7 +2104,10 @@ export class GraphQLService {
 					name: 'directus_fields_meta',
 					fields: Object.values(schema.read.collections['directus_fields'].fields).reduce((acc, field) => {
 						acc[field.field] = {
-							type: field.nullable ? getGraphQLType(field.type) : GraphQLNonNull(getGraphQLType(field.type)),
+							type:
+								field.nullable || field.defaultValue === 'AUTO_INCREMENT'
```

Comment:
We'll have to check for UUID generation as well as auto increment

Reply:
done on  commit 4dd93b9